tests/test_images_conversion.py

```diff
@@ -5,25 +5,66 @@
 import arcus.ml.images.io as ami
 import arcus.ml.images.conversion as conv
 import logging
-import numpy as np 
+import numpy as np
+from hypothesis import given, strategies as st
 
 image_path = 'tests/resources/images/lungs'
 
+def is_black_or_white(cell):
+    return cell == 0 or cell == 1
+def all_black_and_white(arr):
```

Comment:
Some help, I wanted to use `np.all` but seems only to work with 2D arrays.

Reply:
Aha, I think I can fix this with `.all()` which works also for multi-dimensional arrays.

Reply:
I wanted to verify if the result 3D-array only contains 1 or 0 values. But seems that the `np.all` can only work with static ones and 2D-arrays. So, I did the assertion myself, but please correct me if you have something better. 😉